P5/Source/Specs/annotationBlock.xml

```diff
@@ -1,47 +1,48 @@
 <!--
-Copyright TEI Consortium. 
-Dual-licensed under CC-by and BSD2 licences 
-See the file COPYING.txt for details
+    Copyright TEI Consortium. 
+    Dual-licensed under CC-by and BSD2 licences 
+    See the file COPYING.txt for details
 -->
 <?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
 <elementSpec ident="annotationBlock" module="spoken" xmlns="http://www.tei-c.org/ns/1.0">
-    <!-- should this be in spoken module or analysis ? -->
-    <desc xml:lang="en" versionDate="2016-02-27">groups together various annotations, e.g.
-        for parallel interpretations of a spoken segment.</desc>
-    <desc xml:lang="ja" versionDate="2018-12-28">様々な注釈をグループ化する．例えば，話されたセグメントの並行解釈のためなど．</desc>
-    <classes>
-      <memberOf key="model.standOffPart"/>
-      <memberOf key="model.divPart.spoken"/>
-      <memberOf key="att.ascribed"/>
-      <memberOf key="att.timed"/>
-      <memberOf key="att.global"/>
-    </classes>
-    <content>
-      <alternate minOccurs="0" maxOccurs="unbounded">
-        <elementRef key="u"/>
-        <elementRef key="spanGrp"/>
-        <classRef key="model.global.spoken"/>
-      </alternate>
-    </content>
-    <!-- shd there be a constraint to say if no child element is supplied the @corresp attrib must be used -->
-    <exemplum xml:lang="en">
-        <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#NONE">
-            <annotationBlock who="#SPK1" start="#T2" end="#T3" xml:id="ag20">
-                <u xml:id="u20">
-                    <seg xml:id="seg37" type="utterance" subtype="modeless">
-                        <w xml:id="w46">Yeah</w>
-                    </seg>
-                </u>
-            </annotationBlock>
-            <annotationBlock who="#SPK1" start="#T5" end="#T6" xml:id="ag21">
-                <u xml:id="u21">
-                    <seg xml:id="seg38" type="utterance" subtype="modeless">
-                        <w xml:id="w47">Mhm</w>
-                    </seg>
-                </u>
-            </annotationBlock>
-        </egXML>
-    </exemplum>
+  <!-- should this be in spoken module or analysis ? -->
```

Comment:
If this is still a questions, then I think the answer is `spoken`. If it's been resolved, the comment should be removed before merging.

Reply:
It was defined in spoken because spoken needed it for the ISO standard. The ideal placement would imo be in analysis and merely used by spoken.

Reply:
This comment has been there since Lou created the file. Agree with removing it though...

Reply:
Could it, on deletion, be honoured with an issue ticket, perhaps? Simply because `spoken` is not the perfect place for annotationBlock. `analysis` is. 
Surely there's no need to add this to the heap of issues to be solved now, but it's worth bearing in mind for quieter times.
OK to post an issue referencing this set of comments?

Reply:
@bansp I think it should go in as a comment on #1710. I'm not sure how linkable these comments will be after the PR is completed, so maybe don't rely on being able to point here.